consul/aio.py

```diff
@@ -55,17 +70,20 @@ def __del__(self):
                               ResourceWarning)
                 self.close()
 
-    def get(self, callback, path, params=None):
+    def get(self, callback, path, params=None, timeout=None):
         uri = self._uri(path, params)
-        return self._request(callback, 'GET', uri)
+        timeout = timeout if timeout else self.timeout
```

Comment:
What should `timeout = 0` mean here?  If the client has a timeout set, but you want a given function not to timeout, wouldn't you want 0 to disable timeout?